### PR TITLE
Resetting patient based indicator to default upon modelSelection

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## Description

Resetting patient-based indicator to default upon model selection. 
## JIRA Ticket

[MAT-3001](https://jira.cms.gov/browse/MAT-3001)
## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)

- [ ] None applicable

### CODE REVIEW

File: src/main/java/mat/client/measure/ManageMeasurePresenter.java [hunk 1]
```diff
@@ -702,7 +704,6 @@ private void setPatientBasedIndicatorBasedOnScoringChoice(DetailDisplay detailDi
             detailDisplay.getPatientBasedListBox().setSelectedIndex(0);
             detailDisplay.getHelpBlock().setText("Patient based indicator set to no.");
         } else {
-            resetPatientBasedInput(detailDisplay);
             detailDisplay.getHelpBlock().setText("Patient based indicator set to yes.");
```
[the remark this answers is not in the document]
Comment (RohitKandimalla) [reply src/main/java/mat/client/measure/ManageMeasurePresenter.java:707]: Yes, Either we have to reset the options for Patient-based or rewrite them for QDM.

[the remark this answers is not in the document]
Comment (RohitKandimalla) [reply src/main/java/mat/client/measure/ManageMeasurePresenter.java:707]: For FHIR we are removing an item from the list, So I can add an else statement for QDM and add the removed item from the list or reset the entire list and start alone every time. I took the latter approach as it's not expensive.
